### DISCUSSION TITLE
question about week 1 homework - question 5

### DISCUSSION THREAD

**Discussion by alifelman** [LiNk-NY/IntroR#35] [General]
hi everyone, I'm stumbling on question 5 where it asks you "What function do you use to look at the car makes and models?"

It appears to me that the car makes and models are in the first column (the header of each row). I used the unique function (thanks Gemini) to print unique values in row...but it thought I was referring to the next row, which has mpg. 

Does anyone have any idea how to do this? Or even what to ask Gemini? Thanks :)

**Comment by KrisDelaPena** [comment]
Hello! You can actually ask this in R (Help) 
try searching ??rownames in RStudio or Colab to check which functions to use and how to use them. 
Hope this helps!

**Reply by alifelman** [reply]
thank you!! I figured it out :)

**Reply by ralythompson24** [reply]
Thanks @KrisDelaPena !
